src/sources/geniuscom.cpp

```diff
@@ -51,12 +53,13 @@ static std::string remove_chars_for_url(const std::string_view input)
 std::vector<LyricDataRaw> GeniusComSource::search(const LyricSearchParams& params, abort_callback& abort)
 {
     auto request = http_client::get()->create_request("GET");
+    
+    request->add_header("Authorization: Bearer ZTejoT_ojOEasIkT9WrMBhBQOz6eYKK5QULCMECmOhvwqjRZ6WbpamFe3geHnvp3"); //anonymous Android app token

```

Comment:
I created one that we own/control, can we swap it out with that one please?

```suggestion

    // The Genius API client access key for foo_openlyrics
    request->add_header("Authorization: Bearer qQx0MP0GAY-jvPX0aLz_vqBQmy6R9eI6I7MY7XMWhCxT2ixoRDSxRpzcEzhMhO0z");
```